utils/measure/measure.py

```diff
@@ -47,7 +48,11 @@
     POWER_METER_TUYA,
 ]
 
-SELECTED_POWER_METER = config("POWER_METER")
+NOT_DEFINED="Not defined"
+SELECTED_POWER_METER = config("POWER_METER","Not defined")
+if SELECTED_POWER_METER == NOT_DEFINED:
```

Comment:
As far as I remember, config("POWER_METER") results in an exception when .env is missing beofre reaching the main function.
Whatever the method is, it would have been nice to be informed that the .env file is missing rather than getting an exception.